internal/gen/controller.swagger.json

```diff
@@ -6,49 +6,49 @@
   },
   "tags": [
     {
-      "name": "ScopeService"
+      "name": "controller.api.services.v1.ScopeService"
```

Comment:
This change does not appear to affect the rendered API docs: https://boundary-cgn9uklnw-hashicorp.vercel.app/api-docs/account-service